src/lwc/signals/core.ts

```diff
@@ -264,17 +333,21 @@ function $signal<T>(
           setter(newValue);
         }
       },
+      brand: SIGNAL_OBJECT_BRAND,
       readOnly: {
         get value() {
           return getter();
         }
+      },
+      peek() {
+        return _storageOption.get();
       }
     };
 
-  // We don't want to expose the `get` and `set` methods, so
-  // remove before returning
   delete returnValue.get;
```

Comment:
You are about to reach the point were it might make sense to put the properties to remove in an array and delete them in a loop. Just something to consider

Reply:
Yup, I have an issue to fix this ugliness already https://github.com/cesarParra/lwc-signals/issues/27